### PR TITLE
Refactor setting

### PR DESCRIPTION
**Pull request by Bambooin**

### CODE REVIEW

File: res/values-en/strings.xml [hunk 1]
```diff
@@ -52,14 +52,14 @@
     <string name="pref_reset">Reset</string>
     <string name="pref_reset_summary">Use Default Settings to Replace the contents in /sdcard/rime</string>
 
-    <string name="pref_keyboard">Keyboard</string>
+    <string name="pref_preferences">Preferences</string>
```
Comment (osfans) [inline res/values-en/strings.xml]: 想參考谷歌拼音的設置菜單。它是分輸入、鍵盤、字典、其他等幾類。

Comment (Bambooin) [reply res/values-en/strings.xml]: I use android 7.1.1, and I refer the AOSP not the Google input method.
I never install Google input method or use AOSP for privacy, especially I have trime.
The AOSP settting is similar with our sub preferences in Preferences, so I use this.
What about other choice or keep using keyboard???

Comment (Bambooin) [reply res/values-en/strings.xml]: You can refer it in setting->language & input->virtual keyboard->AOSP

File: res/values/strings.xml [hunk 2]
```diff
@@ -41,10 +41,10 @@
     <string name="pref_credits">鳴謝</string>
     <string name="pref_pime">Win10 PIME輸入法平臺</string>
 
-    <string name="pref_advanced">高級設定</string>
+    <string name="pref_advanced">輸入</string>
```
Comment (osfans) [inline res/values/strings.xml]: 這個key要不要改回pref_input？

File: res/values/strings.xml [hunk 3]
```diff
@@ -55,7 +55,7 @@
     <string name="pref_reset">回廠</string>
     <string name="pref_reset_summary">使用預設設定覆蓋/sdcard/rime下同名檔案</string>
 
-    <string name="pref_preferences">偏好設定</string>
+    <string name="pref_preferences">鍵盤</string>
```
Comment (osfans) [inline res/values/strings.xml]: 這個改回pref_keyboard？

File: res/values-en/strings.xml [hunk 4]
```diff
@@ -52,7 +52,7 @@
     <string name="pref_reset">Reset</string>
     <string name="pref_reset_summary">Use Default Settings to Replace the contents in /sdcard/rime</string>
 
-    <string name="pref_preferences">Preferences</string>
+    <string name="pref_keyboard">Preferences</string>
```
Comment (osfans) [inline res/values-en/strings.xml]: 這個要改成Keyboard？

File: res/values-en/strings.xml [hunk 5]
```diff
@@ -38,7 +38,7 @@
     <string name="pref_credits">Credits</string>
     <string name="pref_pime">Win10 PIME Platform</string>
 
-    <string name="pref_advanced">Advanced</string>
+    <string name="pref_input">Advanced</string>
```
Comment (osfans) [inline res/values-en/strings.xml]: 這個要改成Input？

File: res/values-en/strings.xml (hunk 4 above)
Comment (Bambooin) [reply res/values-en/strings.xml]: What I did(order and translate) is referring AOSP.
It has similarity with trime in preference(keyborad).

aosp
pre

Comment (osfans) [reply res/values-en/strings.xml]: 用哪一種比較好呢？google pinyin還是AOSP呢？

Comment (Bambooin) [reply res/values-en/strings.xml]: Google pinyin is better for Chinese people, as it is more popular than AOSP(except Sougou).

Comment (osfans) [reply res/values-en/strings.xml]: 好，那就參考谷歌拼音了。

**Pull request merged**